src/muse/readers/csv.py

```diff
@@ -129,7 +133,7 @@ def read_technodata_timeslices(filename: Union[Text, Path]) -> xr.Dataset:
     data = csv[csv.technology != "Unit"]
 
     data = data.apply(lambda x: pd.to_numeric(x, errors="ignore"))
-    data = check_utilization_not_all_zero(data, filename)
+    check_utilization_not_all_zero(data, filename)
```

Comment:
in principle, the `technodata_timeslices` file also has an `utilization_factor` column that will need validating. See https://github.com/EnergySystemsModellingLab/MUSE_OS/blob/enforce_minimum_service_factor_limits/src/muse/data/example/default_timeslice/technodata/power/TechnodataTimeslices.csv